### DISCUSSION TITLE
Solve a routing problem with fixed starting point and algorithm to autodiscover the last point

### DISCUSSION THREAD

**Discussion by kurupam** [google/or-tools#4472] [Routing (and legacy CP) questions]
Hi,

I have the below problem,

Suppose starting location is 'A' and I have go through set of locations (B,C,D) and follow the shortest path, using RoutingIndexManager provided by ortools. But, I don't want to put end index manually, but would need Routingmanager to find the shortest path and autodiscover the last destination. Plus, I don't want start index = end index scenario, which means I don't intend the journey to end at start index. I tried option of depot, endindex as null or empty, but nothing succeeded.

Can some expert from ORTools team, point me on how to to achieve this using ortools?

**Comment by lperron** [comment]
asked once per week.

Just tweak you distance matrix to have all are going to the depot have a null distance.
